examples/openshift/openshift-values.yaml

```diff
@@ -13,40 +11,30 @@ spire-server:
       repository: spiffe/tornjak-backend
       tag: ubi-v1.4.1
     ingress:
-      enabled: true
```

Comment:
Disable the ingresses by default as most users wont want to enable them all. We could maybe add a  mixin values example that sets enabled on them all.